terra-notebooks-playground/README.md

```diff
@@ -1,3 +1,10 @@
+**IMPORTANT NOTE FOR USERS NEW TO JUPYTER NOTEBOOKS:**
+
+* *Setup* notebooks are used to set up computational environments that are language-specific. This means that if a workspace contains both Python and R-based notebooks, and has *Setup* notebooks to run beforehand, users must run the appropriate *Setup* notebook before running subsequent notebooks *in that language*. In other words, if you run the ***R Environment Setup*** notebook in this workspace, and then attempt to run the Python-based notebooks, it will not work. To fix this, you will need to run the ***Python Environment Setup*** notebook within this cluster first.
```

Comment:
This is helpful, but it is also covered further down.  Should this blurb now be deleted? 

> 'It is important that the “environment setup” notebooks listed at the top of each category (bold) be run before any other notebooks in their respective categories. The “environment setup” notebooks perform important initializing steps (enabling extensions, etc) that are necessary for the functions in the subsequent notebooks. If you are entirely new to Jupyter and are not sure where to start, try the first two notebooks at the top of each list (italics)!'

Reply:
Deleted a sentence in the middle of that paragraph (the rest of the instructions in the paragraph refer specifically to the list right below this paragraph, so I would rather keep the part of the instruction that tells you which specific notebooks to run, whereas the note at the top is more of a general note about the concept of a setup notebook)